src/sass/components/_tour.scss

```diff
@@ -37,12 +37,40 @@
 
 .popover-wrapper {
   position: absolute;
-  top: -10px;
+  top: -12px;
```

Comment:
![zindex](https://user-images.githubusercontent.com/1581694/119443556-6b663c80-bcef-11eb-925f-34b16b06e781.jpg) 

If we bump this z-index to at least 1, it won't lay behind this table cell neighbor I believe

Reply:
fixed